dags/utils/dataproc.py

```diff
@@ -29,6 +29,7 @@ def __init__(self,
                  num_preemptible_workers=0,
                  service_account=None,
                  artifact_bucket='moz-fx-data-prod-airflow-dataproc-artifacts',
+                 init_actions_uris=None,
```

Comment:
Semantically this is a bit confusing but I suppose the purpose is to differentiate between setting a default value based on `artifact_bucket` and explicitly setting no init actions via `[]`. It also appears to match the upstream.

Reply:
I agree, this is confusing that there is a default bootstrap when you don't explicitly set one (and this default to `None`).

Reply:
currently nobody is setting the artifact_bucket parameter. Would you prefer I remove it as a parameter entirely? And if init_action_uris is None, set it to the default init script(within that bucket). Otherwise they can set it to [].

Reply:
The only other place it is currently used is to point to the scriptrunner jar and init scripts that mimic the custom emrsparkoperator functionality

Reply:
removed artifact_bucket as a param

Reply:
the confusion is likely because a previous pr was using artifact_bucket=None to set no init actions rather than setting init actions itself (which didnt exist as a param before)